ddl/ddl_test.go

```diff
@@ -94,14 +98,6 @@ func checkHistoryJobArgs(c *C, ctx context.Context, id int64, args *historyJobAr
 	if args.db != nil && len(args.tblIDs) == 0 {
 		return
 	}
-	// only for dropping schema job
```

Comment:
Because the history job is DDL job, its RawArgs is covered by Args when doing encode. So it doesn't have the RawArgs value now.